test/navbar-components/navigationBarTest.js

```diff
@@ -379,10 +379,8 @@ describe('navigationBar tests', function () {
           this.app.client
             .activateURLMode()
             .waitForExist(urlBarIcon)
-
-            // TODO: Fix
             .getAttribute(urlBarIcon, 'class').then((classes) =>
-              classes.includes('fa-unlock') && classes.includes('insecure-color')
+              classes.includes('fa-unlock')
```

Comment:
I'm not sure if this is a proper way since it is important to confirm that `fa-unlock` does actually have the red color, not grey or green one, and that seems the main object of this test including ones below.

Is it impossible to replace these `.getAttribute` block with something which uses data-test-id, such as `[data-test-id="fa-unlock"][data-test-id="insecure-color"]`?